.gitattributes

```diff
@@ -0,0 +1,4 @@
+internal/mocks/*.go linguist-generated=true
```

Comment:
FYI so that when changing these, the "View PR" view on Github says that the files are automatically generated and people can just mark "viewed" on them